e2e/src/completion/completion_macros.spec.ts

```diff
@@ -12,20 +12,17 @@ suite('Should suggest macros completions', () => {
     ['extract_last_name', 'extract_last_name'],
   ];
 
-  // TODO: we should rework this test
-  test.skip('Should suggest macros', async () => {
+  test('Should suggest macros', async () => {
     // arrange
     const docUri = getCustomDocUri(PROJECT_FILE_NAME);
     await activateAndWait(docUri);
+    await waitManifestJson(PROJECT);
 
     // act
-
-    // new Position(0, 17) -- ?
-    const actualCompletionList = await triggerCompletion(docUri, new Position(0, 15), 'e');
+    const actualCompletionList = await triggerCompletion(docUri, new Position(0, 17));
 
     // assert
     const expectedCompletions = getMacrosCompletionList();
-    assertThat(actualCompletionList.items.length, expectedCompletions.length);
```

Comment:
The problem of this test also that if we return `undefined`, the default set of suggestions contains expected values `extract_first_name` and `extract_last_name`. Can we make it more specific somehow?

Reply:
Completion provider returns items with detail property equal to 'Macro'. Added detail property comparison.

Reply:
The same for other autocompletion tests